src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1918,8 +1928,12 @@ public synchronized void scheduled(ChangeBasedEvent event, ParametersAction para
          *            The event that originally triggered the build.
          */
         private void cancelJob(GerritTriggeredEvent event) {
+            if (!(job instanceof Queue.Task)) {
+                logger.error("Error canceling job. The job is not of type Task.");
+            }
```

Comment:
I guess you should return directly after this?

And please log the task as well, so the admin knows which of the 10k jobs the log is talking about :)